app/src/main/java/cl/maleb/testcarsales/ui/covid/CovidFragment.kt

```diff
@@ -0,0 +1,112 @@
+package cl.maleb.testcarsales.ui.covid
+
+import android.app.DatePickerDialog
+import android.os.Bundle
+import android.view.LayoutInflater
+import androidx.lifecycle.Observer
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.ViewModelProvider
+import cl.maleb.testcarsales.data.Result
+import cl.maleb.testcarsales.databinding.CovidLayoutBinding
+import cl.maleb.testcarsales.di.Injectable
+import cl.maleb.testcarsales.di.injectViewModel
+import java.util.*
+import javax.inject.Inject
+
+class CovidFragment : Fragment(), Injectable {
+    @Inject
+    lateinit var viewModelFactory: ViewModelProvider.Factory
+
+    private lateinit var viewModel: CovidViewModel
+    private lateinit var binding: CovidLayoutBinding
+
+    // Calendar variables
+    val c = Calendar.getInstance()
+    val year = c.get(Calendar.YEAR)
+    val month = c.get(Calendar.MONTH)
+    val day = c.get(Calendar.DAY_OF_MONTH)
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+
+    }
+
+    override fun onCreateView(
```

Comment:
All this code I would pass to onViewCreated

Reply:
Do you mean ALL of the onCreateView code?

Reply:
@maleb1337 yes, I can't see any specific task you may need to do _before_ creating the View

Reply:
Yup. Now I removed onCreateView, now only I have onViewCreated with some adaptions regarding the layout binding.